profiles/realtime-virtual-host/script.sh

```diff
@@ -98,12 +98,13 @@ start() {
         echo `cat $CACHE_VALUE_FILE` > $KVM_LAPIC_FILE
     fi
     systemctl start rt-entsk
-
+    irqbalance_banned_cpus_setup $TUNED_isolated_cpumask
```

Comment:
Same here, enclose `$TUNED_isolated_cpumask` in double quotes.

Reply:
Sure thing, I'll re-push soon.  Thanks for the quick reviews!